html/includes/table/syslog.inc.php

```diff
@@ -75,5 +77,6 @@
     'rowCount' => $rowCount,
     'rows'     => $response,
     'total'    => $total,
+'sql' => $sql,
```

Comment:
Looks like you've left this in from debugging. Can you remove please :)

Also, you can use logfile($sql); and it will write to logs/librenms.log if you need to.